README.md

```diff
@@ -130,6 +139,11 @@ Puppet nodes, including:
 - [Using Conjur host factory](#conjur-host-factory)
 - [Using pre-established host identities (**Conjur Enterprise v4 only**)](#pre-established-host-identity-conjur-enterprise-v4-only)
 
+**Note that establishment of identity using host factory tokens through this module is
+deprecated and will be removed in the next major version. If you are still using this
+feature, please use the [v2](https://github.com/cyberark/conjur-puppet/tree/v2) branch
++of this project or a release version `<3.0.0`**
```

Comment:
```suggestion
+of this project or a release version `<3.0.0`.**
```
Also, instead of repeating this text in several places, what about putting a deprecations section below system requirements and referencing as needed?

Finally, maybe repeat here that HFTs can still be used to create hosts used with this module, but these identities need to be established outside of the module?